connectors/s3-connector/src/main/scala/zio/connect/s3/S3Connector.scala

```diff
@@ -1,59 +1,105 @@
 package zio.connect.s3
 
 import zio.connect.s3.S3Connector._
-import zio.prelude.Subtype
+import zio.prelude.{Newtype, Subtype}
 import zio.stream.{ZSink, ZStream}
 import zio.{Trace, ZIO}
 
+import scala.language.implicitConversions
 trait S3Connector {
+  def copyObject(region: => Region)(implicit
+    trace: Trace
+  ): ZSink[Any, S3Exception, CopyObject, CopyObject, Unit]
```

Comment:
Can't this be implemented here in term of the other copyObject?
`
final def copyObject(region: => Region)(implicit
     trace: Trace
   ): ZSink[Any, S3Exception, CopyObject, CopyObject, Unit] = 
   copyObject(region, region)
`